src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -574,6 +574,62 @@ class C
             End Using
         End Function
 
+        <WpfFact>
+        Public Async Function TestNonTrailingNamedArgumentInCSharp7_1() As Task
```

Comment:
This kind of test seems more natural, as I want to check that results from other providers is not suppressed (the named argument provider does set the `IsExclusive` flag on the results it provides).